src/check.js

```diff
@@ -0,0 +1,123 @@
+/* eslint no-use-before-define: ["error", { "variables": false }] */
+
+function updateCompletedListArray() {
+  completedTasks = [];
+
+  taskList.forEach((task) => {
+    if (task.done) completedTasks.push(`${taskList.indexOf(task)}`);
+  });
+}
+
+function saveLocalList() {
+  localStorage.setItem('taskList', JSON.stringify(taskList));
+}
+
+function toggleChecked(e) {
+  const checkStatus = e.target.checked;
```

Comment:
- [ ] Please change all regular functions to es6 arrow function syntax. Following the JavaScript best practices guide.